lib/membrane_ice_plugin/ice_endpoint.ex

```diff
@@ -136,34 +138,40 @@ defmodule Membrane.ICE.Endpoint do
                 [
                   magic: nil,
                   in_nominated_pair: false,
-                  passed_check_from_browser: false,
-                  passed_check_from_sfu: false
+                  passed_check_from_browser: false
                 ]
   end
 
   @impl true
   def handle_init(options) do
     %__MODULE__{
-      ice_lite?: ice_lite?,
       integrated_turn_options: integrated_turn_options,
       dtls?: dtls?,
       handshake_opts: hsk_opts
     } = options
 
-    if ice_lite?, do: Process.send_after(self(), :maybe_send_binding_indication, 1000)
+    Process.send_after(self(), :maybe_send_keepalive, @time_between_keepalives)
 
-    {:ok,
-     %{
-       turn_allocs: %{},
-       integrated_turn_options: integrated_turn_options,
-       fake_candidate_ip: integrated_turn_options[:mock_ip] || integrated_turn_options[:ip],
-       selected_alloc: nil,
-       dtls?: dtls?,
-       hsk_opts: hsk_opts,
-       component_connected?: false,
-       cached_hsk_packets: nil,
-       component_ready?: false
-     }}
+    state =
+      %{
+        id: to_string(Enum.map(1..10, fn _ -> Enum.random(?a..?z) end)),
+        turn_allocs: %{},
+        integrated_turn_options: integrated_turn_options,
+        fake_candidate_ip: integrated_turn_options[:mock_ip] || integrated_turn_options[:ip],
+        selected_alloc: nil,
+        dtls?: dtls?,
+        hsk_opts: hsk_opts,
+        component_connected?: false,
+        cached_hsk_packets: nil,
+        component_ready?: false,
+        pending_connection_ready?: false,
+        connection_status_sent?: false,
+        sdp_offer_arrived?: false,
+        ice_restart_timer: nil
+      }
+      |> start_ice_restart_timer()
```

Comment:
Shouldn't this timer be set only when we receive new remote crednetials or first remote candidate?

Reply:
Currently timer is started after receiving `:ice_restart` and in `handle_init`, but I think that moving it to `handle_prepared_to_playing` would be good idea